lib/coverband.rb

```diff
@@ -6,6 +6,7 @@
 require "coverband/version"
 require "coverband/at_exit"
 require "coverband/configuration"
+require "coverband/utils/dead_methods"
```

Comment:
do we want these just conditional required in the rake task, etc... since it only supports later Ruby methods wouldn't trying to call refine cause issues?